src/test/java/AirlockMessageGenerator.java

```diff
@@ -5,12 +5,17 @@
 import java.util.ArrayList;
 
 class AirlockMessageGenerator {
+
     static AirlockMessage generateAirlockMessage() {
+        return generateAirlockMessage("routing-key");
+    }
+
+    static AirlockMessage generateAirlockMessage(String routingKey) {
         long ts = System.currentTimeMillis();
         AirlockMessage message = new AirlockMessage();
         for (int i = 1; i <= 3; i++) {
             EventGroup eventGroup = new EventGroup();
-            eventGroup.eventRoutingKey = "routing-key-" + Integer.toString(i);
```

Comment:
Is it designed as to use a _dot_ instead of a _hyphen_? (I mean, it is not a typo)

Reply:
Hyphen is not a separator. This is a part of token. Dot is added to use wildcard for routing key at tests.